include/wlc/defines.h

```diff
@@ -12,11 +12,13 @@ extern "C" {
 #  define WLC_NONULLV(...) __attribute__((nonnull(__VA_ARGS__)))
 #  define WLC_PURE __attribute__((pure))
 #  define WLC_CONST __attribute__((const))
+#  define WLC_UNUSED __attribute__((unused))
```

Comment:
Instead of __attribute__((unused)) wlc has used to (void)foobar; to ignore "not used" warnings.